index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en" ng-app>
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
+</head>
+<body>
+  {{ "Hello" + " World" }}
```

Comment:
this is a practice comment, to show how to make comments when doing a code review. This is my PR, so I don't need to make a real comment. :ice_cream: